.travis.yml

```diff
@@ -3,26 +3,34 @@ sudo: false
 branches:
   only:
     - master
-python:
-  - 3.5
-  - 3.6
-
-env: 
-    - PYSAL_PYPI=true PYSAL_PLUS=true
-    - PYSAL_PYPI=true PYSAL_PLUS=false
-    - PYSAL_PYPI=false PYSAL_PLUS=true
-    - PYSAL_PYPI=false PYSAL_PLUS=false
 
 matrix:
+  include:
```

Comment:
`include` can be deleted since the environments defined by the travis build matrix (`python` and `env`) will kick in. I've tried on [my fork](https://github.com/weikang9009/spaghetti/blob/python37/.travis.yml) and it [works fine](https://travis-ci.org/weikang9009/spaghetti)

Reply:
@weikang9009 got it up and running here! +1 thanks!